src/RequestBuilder.php

```diff
@@ -126,7 +127,7 @@ public function build($resource, $method, array $options = [])
             $action['httpMethod'],
             $uri,
             ['Content-Type' => 'application/json'],
-            $body ? json_encode($body) : null
+            $body ? GuzzleHttp\json_encode($body) : null
```

Comment:
Additionally, since `GuzzleHttp\json_decode` (or a replacement) throws an exception, the ternary is ineffective.